components/root/root.tsx

```diff
@@ -136,14 +141,15 @@ function LoggedInRoute<T>(props: LoggedInRouteProps<T>) {
     );
 }
 
-const noop = () => {}; // eslint-disable-line no-empty-function
+const noop = () => {
+}; // eslint-disable-line no-empty-function
```

Comment:
hmm.. the import shifting is problem that happens when I use the 'fix eslint problems' option from my intellij idea IDE. I had a [thread](https://community.mattermost.com/private-core/pl/h3478x75npbabn6jroo17tht5a) about it a few months back in the Growth channel and Julien said he also faced the same issue in VScode. I guess, I will have to keep in mind to manually arrange the imports for now.
About the other lint issues (spaces around objects and semicolons after arrow function) I am not sure how they were introduced. I reverted those changes and used the usual 'fix eslint problem' and it did not put spaces or semicolons back. I will keep an eye and try to see if, how and when these changes appear in the future. 
My IDE lint settings looks like
![Screenshot 2023-01-14 at 14 06 37](https://user-images.githubusercontent.com/841955/212473305-f77e07cd-0b7b-4f99-94b2-99a50d5e8b1e.png)
I do not use any other plugin or have any other lint settings other than that.

Reply:
after further looking into import arrangement I found out that in certain cases the import is resolved from the project source code instead of node_modules. For instance, for
```
import {NotifyAdminRequest} from '@mattermost/types/cloud';
import {NotifyStatus, NotifyStatusValues, useGetNotifyAdmin} from 'components/common/hooks/useGetNotifyAdmin';
```
The IDE shows a req squiggle and suggests rearrangment because it finds the `NotifyAdminRequest` from `packages/types/src/cloud.ts` instead of node_modules